ui/src/ensembler/components/table/ListRouterVersionsForEnsemblerTable.js

```diff
@@ -115,7 +118,7 @@ export const ListRouterVersionsForEnsemblerTable = ({
     </EuiCallOut>
   ) : (
     <Fragment>
-      {ensemblerUsedByCurrentRouterVersion ? (
+      {(ensemblerUsedByCurrentRouterVersion && results.totalActiveCount === 0) ? (
```

Comment:
This condition has been added to ensure that the active router versions using an ensembler are listed first (if there are any) instead of listing the current router versions. This is because all active router versions are current router versions but not the other way round. 

In order to ensure that the UI displays these tables in this order of priority, the new condition is required to ensure that the current router versions are only listed when there are NO active router versions.